source/util/SGM/SGMSettings.cpp

```diff
@@ -44,7 +44,7 @@ void SGMSettings::load(){
 #ifdef Q_WS_MAC
 	SGMDataFolder_ = settings.value("SGMDataFolder", "/Users/fawkes/Documents/CLS/SGM/ACQUAMANDATA").toString();
 #else
-	SGMDataFolder_ = settings.value("SGMDataFolder", "/home/sgm/beamline/programming").toString();
+	SGMDataFolder_ = settings.value("SGMDataFolder", "/home/workmai/beamline/programming").toString();
```

Comment:
Yeah, this is one of those changes that I have to make from my machine each time I pull down SGM and wish to run it. I usually revert this file before I push it back up to the remote. Must have forgotten this time. It'll get changed back before merging.